### PR TITLE
Revamp update-bridge workflow.

### PR DESCRIPTION
**Pull request by jkodroff**

### CODE REVIEW

**Comment by jkodroff** [comment]
@stack72 See #35 for output.

File: .github/workflows/update-bridge.yml [hunk 1]
```diff
@@ -93,6 +86,13 @@ jobs:
         pythonversion:
         - "3.7"
 "on":
-  repository_dispatch:
-    types:
-    - update-bridge
+  workflow_dispatch:
```
Comment (stack72) [inline .github/workflows/update-bridge.yml:83]: this must also be a repostory_dispatch as we will be calling this remotely from the other repo

File: .github/workflows/update-bridge.yml [hunk 2]
```diff
@@ -52,35 +47,33 @@ jobs:
...
-        pr_body: '*Automated PR*'
-        pr_label: automation/merge
-        pr_reviewer: stack72
-        pr_title: Upgrade to ${{ github.event.client_payload.ref }} of pulumi-terraform-bridge
-        source_branch: update-bridge/${{ github.event.client_payload.ref }}-${{ github.run_id
-          }}
+        pr_allow_empty: false
+        pr_body: This pull request was generated automatically by the update-bridge workflow in this repository.
+        pr_label: impact/no-changelog-required
+        pr_reviewer: jkodroff
```
Comment (stack72) [inline .github/workflows/update-bridge.yml]: this should be the team right?

File: .github/workflows/update-bridge.yml [hunk 3]
```diff
@@ -52,35 +47,33 @@ jobs:
...
+    - name: Update Pulumi SDK (provider/go.mod)
+      run: cd provider && go mod edit -require github.com/pulumi/pulumi/sdk/v3@v${{ github.event.inputs.sdk_version }} && go mod tidy
+    - name: Update Pulumi SDK (sdk/go.mod)
+      run: cd sdk && go mod edit -require github.com/pulumi/pulumi/sdk/v3@v${{ github.event.inputs.sdk_version }} && go mod tidy
+    - run: make tfgen
     - run: make build_sdks
-    - run: chg add "Upgrading pulumi-terraform-bridge to ${{ github.event.client_payload.ref
-        }}
     - name: commit changes
       uses: EndBug/add-and-commit@v7
```
Comment (stack72) [inline .github/workflows/update-bridge.yml]: We should be able to combine these 2 steps using `https://github.com/peter-evans/create-pull-request`

File: .github/workflows/update-bridge.yml [hunk 4]
```diff
@@ -52,35 +47,33 @@ jobs:
       with:
         python-version: ${{matrix.pythonversion}}
     - run: sudo npm install -g chg
```
Comment (stack72) [inline .github/workflows/update-bridge.yml]: we don't need this dep installed anymore

File: .github/workflows/update-bridge.yml [hunk 5]
```diff
@@ -1,31 +1,26 @@
...
-  PULUMI_LOCAL_NUGET: ${{ github.workspace }}/nuget
-  PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-  TRAVIS_OS_NAME: linux
-  YC_CLOUD_ID: ${{ secrets.YC_CLOUD_ID }}
-  YC_FOLDER_ID: ${{ secrets.YC_FOLDER_ID }}
-  YC_SERVICE_ACCOUNT_KEY_FILE: ${{ secrets.YC_SERVICE_ACCOUNT_KEY_FILE }}
+  GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+  # NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  # NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (stack72) [inline .github/workflows/update-bridge.yml:5]: Just for reference, these will carry across when we put this in CI Mgmt by default - but that's no issue for now

**Review by stack72 — COMMENTED** [review]
This looks good - a few small comments more than anything - the only must change is to allow repository_dispatch as well

File: .github/workflows/update-bridge.yml (hunk 1 above)
Comment (jkodroff) [reply .github/workflows/update-bridge.yml:83]: I plan to use https://github.com/marketplace/actions/workflow-dispatch or to run `gh workflow run` to trigger from another repo if that's acceptable.

File: .github/workflows/update-bridge.yml (hunk 3 above)
Comment (jkodroff) [reply .github/workflows/update-bridge.yml]: Fixed.

File: .github/workflows/update-bridge.yml (hunk 4 above)
Comment (jkodroff) [reply .github/workflows/update-bridge.yml]: Fixed.

File: .github/workflows/update-bridge.yml (hunk 5 above)
Comment (jkodroff) [reply .github/workflows/update-bridge.yml:5]: I may want to ask if we can change that behavior later, but understood.

File: .github/workflows/update-bridge.yml (hunk 2 above)
Comment (jkodroff) [reply .github/workflows/update-bridge.yml]: Fixed.

File: .github/workflows/update-bridge.yml (hunk 1 above)
Comment (stack72) [reply .github/workflows/update-bridge.yml:83]: I personally still don't see why we are insistent on changing this workflow but 🤷

**Review by stack72 — APPROVED** [review]

**Pull request merged**